src/main/java/ua/com/juja/microservices/keepers/slackbot/model/SlackParsedCommand.java

```diff
@@ -61,4 +66,15 @@ public String getText() {
     public int getUserCountInText() {
         return usersInText.size();
     }
+
+    private String getProperty(String propertyFile, String propertyName) {
```

Comment:
and I think we should do the method static and move to a separate class for reuse this code in another classes.